### DISCUSSION TITLE
Why can't I open the software?

### DISCUSSION THREAD

**Discussion by lidali233** [Griefed/ServerPackCreator#529] [Q&A]
Why does my software get stuck at 20% when I open it?
![image](https://github.com/user-attachments/assets/b2ab2e6a-9200-4c22-b597-e8a6ceb414d6)

**Comment by lidali233** [comment]
[Uploading serverpackcreator.log…]()

**Reply by Griefed** [reply]
Heyo, it appears the upload didn't quite work. There's no log.

Please provide logs as well as console output, if available :)
